src/qutip_jax/ode.py

```diff
@@ -22,6 +22,14 @@ def _float2cplx(arr):
     return arr[0] + 1j * arr[1]
 
 
+@jax.jit
+def dstate(t, y, args):
+    state = _float2cplx(y)
+    H, = args
+    d_state = H.matmul_data(t, JaxArray(state))
+    return _cplx2float(d_state._jxa)
```

Comment:
just a minor thing, does the new complex support in diffrax essentially do the same thing?  though given all the warnings using complex in diffrax directly throws out, maybe better to stick with this manual treatment at the moment.

Reply:
No sure, I did not plan too much with it yet.